src/mutation-observer.js

```diff
@@ -100,13 +100,13 @@ MutationObserver.fixIe = function () {
   }
 
   // We have to call the old innerHTML getter and setter.
-  var oldInnerHtml = Object.getOwnPropertyDescriptor(elementPrototype, 'innerHTML');
```

Comment:
It also seems that there isn't a standard naming convention: document.getElementById.